lib/appcues_flutter.dart

```diff
@@ -82,7 +83,11 @@ class Appcues {
     await _methodChannel.invokeMethod('initialize', {
       'accountId': accountId,
       'applicationId': applicationId,
-      'options': nativeOptions
+      'options': nativeOptions,
+      'additionalAutoProperties': <String, Object> {
+        '_applicationFramework': 'flutter',
+        '_dartVersion': Platform.version
+      }
```

Comment:
the actual Flutter sdk version [does not appear to be programmatically available](https://github.com/flutter/flutter/issues/61814), but the underlying Dart version is, so using that for now to give a little more context in our analytics.